Source/ACE/Entity/WorldObject.cs

```diff
@@ -372,50 +369,200 @@ public float Workmanship
+        {
+            get { return AceObject.Parent; }
+            set { AceObject.Parent = value; }
+        }
+
+        public uint? ParentLocation
+        {
+            get { return AceObject.ParentLocation; }
+            set { AceObject.ParentLocation = value; }
+        }
 
-        // these are all related
         public EquipMask? EquipperPhysicsDescriptionFlag;
-        private readonly List<EquippedItem> children = new List<EquippedItem>();
-
-        public float? ObjScale;
-        public float? Friction;
-        public float? Elasticity;
-        public uint? AnimationFrame;
-        public AceVector3 Acceleration;
-        public float? Translucency;
+
+        public List<EquippedItem> Children { get; } = new List<EquippedItem>();
+
+        public float? ObjScale
+        {
+            get { return AceObject.DefaultScale; }
+            set { AceObject.DefaultScale = value; }
+        }
+
+        public float? Friction
+        {
+            get { return AceObject.Friction; }
+            set { AceObject.Friction = value; }
+        }
+
+        public float? Elasticity
+        {
+            get { return AceObject.Elasticity; }
+            set { AceObject.Elasticity = value; }
+        }
+
+        public uint? AnimationFrame
+        {
+            get { return AceObject.AnimationFrameId; }
+            set { AceObject.AnimationFrameId = value; }
+        }
+
+        public AceVector3 Acceleration { get; set; }
+
+        public float? Translucency
+        {
+            get { return AceObject.Translucency; }
+            set { AceObject.Translucency = value; }
+        }
+
         public AceVector3 Velocity = null;
+
         public AceVector3 Omega = null;
 
         private MotionState currentMotionState;
+
         public MotionState CurrentMotionState
         {
             get { return currentMotionState; }
             set { currentMotionState = value; }
         }
 
-        public uint? DefaultScript;
-        public float? DefaultScriptIntensity;
+        public uint? DefaultScript
+        {
+            get { return AceObject.DefaultScript; }
+            set { AceObject.DefaultScript = value; }
+        }
+
+        public float? DefaultScriptIntensity
+        {
+            get { return AceObject.PhysicsScriptIntensity; }
+            set { AceObject.PhysicsScriptIntensity = value; }
+        }
+
+        // START of Logical Model Data
+
+        public uint? PaletteGuid
+        {
+            get { return AceObject.PaletteId; }
+            set { AceObject.PaletteId = value; }
+        }
+
+        public uint? ClothingBase
+        {
+            get { return AceObject.ClothingBase; }
+            set { AceObject.ClothingBase = value; }
+        }
+
+        private readonly List<ModelPalette> modelPalettes = new List<ModelPalette>();
+
+        private readonly List<ModelTexture> modelTextures = new List<ModelTexture>();
+
+        private readonly List<Model> models = new List<Model>();
```

Comment:
Its a reasonable decision, its just different than we are storing some of our other data.  I dislike inconsistency, but I wont hold up this PR over figuring out the perfect way to do it (if such a thing exists).